### PR TITLE
Output coloring

### PR DESCRIPTION
**Pull request by pfriedrich**
Hi,
Change list:

- Pytest output is now colored (if option is selected)
- Verbose option is now available from context menu

Feel free to comment if you are not happy with some of the changes.
Br,
@simonprn @pfriedrich

### CODE REVIEW

File: lib/atom-python-test-view.coffee [hunk 1]
```diff
@@ -15,14 +15,61 @@ module.exports =
...
+      return new_line
+
+    # TODO: add empty line after "collected" and before "FAILURES/x passed in"
+    addLine: (lines, do_coloring=false) ->
+      for line in lines.split("\n")
+        if line == ""
+          continue
+
+        if do_coloring
+          @message = @colorLine(line)
```
Comment (pghilardi) [inline lib/atom-python-test-view.coffee:62]: With this change only the last line (the overall test status) is being colored. I think that a better behavior would be to color each test output when the --verbose option is selected. Probably you tried to achieve this, but I checked your changes and it doesn`t work as expected, coloring only the status line. 

This is the output that I'm receiving. Is this the expected behavior?

output

Comment (pghilardi) [reply lib/atom-python-test-view.coffee:62]: Maybe you should change the == comparison by a indexOf:
`parts[1].indexOf("FAILED") > -1`

@pfriedrich , what do you think?

Comment (pfriedrich) [reply lib/atom-python-test-view.coffee:62]: Hi,
First of all, thanks for checking our code!

Now, about changing the comparison, what would be the use case for your
solution?
What we had in mind was the "test_script::test_function FAILED" lines in
the verbose output and in that case parts[1] can only be "FAILED", "PASSED"
(and "SKIPPED" which we don't handle yet) and in this case, I feel that an
exact matching is less error prone.
What do you think?
Regards,
Peter Friedrich

2017-02-07 22:22 GMT+01:00 Pedro Ghilardi <notifications@github.com>:

Comment (pghilardi) [reply lib/atom-python-test-view.coffee:62]: I agree with you that an exact matching is less error prone, but when testing the plug-in with your branch I cannot see the strings "FAILED", "PASSED" being colored.

Could you test it again?

Comment (pghilardi) [reply lib/atom-python-test-view.coffee:62]: @pfriedrich ,

Did you tested the code again?

Regards,

**Review by pghilardi — COMMENTED** [review]
@pfriedrich ,

I have tested the code again and detected the error. The color will not be applied when I enable the color output with -s from pytest and I have at least a print output. I am merging the PR anyway because this is minor, but I will improve the test output as soon as possible. Thanks for the contribution!

**Pull request merged**